stable/jenkins/templates/_helpers.tpl

```diff
@@ -190,6 +190,8 @@ Returns kubernetes pod template configuration as code
     resourceLimitMemory: {{.Values.agent.resources.limits.memory}}
     resourceRequestCpu: {{.Values.agent.resources.requests.cpu}}
     resourceRequestMemory: {{.Values.agent.resources.requests.memory}}
+    runAsUser: {{ .Values.agent.runAsUser }}
+    runAsGroup: {{ .Values.agent.runAsGroup }}
```

Comment:
any reason these are configured for the container rather than the pod? unless there's a specific use case, I'd prefer they be at the pod level.

Reply:
Hey, I'm using custom agent sidecar image with some custom default user `example` which doesn't have proper permissions to use mounted jenkins workspace that's  managed by `jnlp` container. 
So in my case I only need to modify them at container level and I'm not sure if there is a need to set it at pod level. Only example I can think of is when you want to also change user in `jnlp` sidecar that's added to agent pod when you overwrite `sideContainerName` and set it to one value for all containers.

Reply:
Yeah, setting these options at the container level does seem like the more common use case.